CommonTools/MVAUtils/interface/GBRForestTools.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CommonTools_MVAUtils_GBRForestTools_h
+#define CommonTools_MVAUtils_GBRForestTools_h
+
+//--------------------------------------------------------------------------------------------------
+//
+// GRBForestToolsTinyXML
```

Comment:
a stray ```TinyXML```.
Perhaps it's "close enough" and an update is not really needed.

Reply:
First I do automatic find and replace and it messes up header guards without me noticing. Then, I avoid doing find a replace to not get any unexpected errors and I get of this. as you said, "close enough"... I could change it but now that the tests are running already let's just stay with it? It's just a backport after all.